src/main/java/step4/RacingRule.java

```diff
@@ -0,0 +1,34 @@
+package step4;
+
+import java.util.List;
+import java.util.Random;
+
+public class RacingRule implements Rule {
+    private static final int CAN_GO = 4;
+    private static final Random random = new Random();
```

Comment:
이 부분도 코드컨벤션이 적용되면 좋겠습니다 😄 
상수 값이니 대문자로 변경 부탁 드립니다 🙇 
```suggestion
    private static final Random RANDOM = new Random();
```